scripts/smeshr/MODIS-STC/MODIS_albedo_basin.sh

```diff
@@ -25,10 +25,13 @@ extract_modis() {
   fi
 }
 
-#BASIN name
+# Basin name and processing parameters
 BASIN='yampa'
-#BASIN_EXTENT='315900.0 4280850.0 348700.0 4322700.0' #East River
-# BASIN_EXTENT='156776.4 4530442.0 365276.4 4426242.0' #yampa
+BASIN_EXTENT='156776.4 4530442.0 365276.4 4426242.0' #yampa
+RES=100
+EPSG='EPSG:32613'
```

Comment:
Thinking out loud and to make this a pure general script, you could pass the four pieces (line 29-32) of information as script arguments

Reply:
Noted! If we end up overhauling this, I think this can be updated. Right now, I leave this hardcoded for an update from this prep script https://github.com/UofU-Cryosphere/isnoda/blob/364dd687155b65bed6fdb32d3c299549dbb060a2/scripts/smeshr/prepare_MODIS_HRRR_inputs.sh#L170C1-L201C72